daemon/update.go

```diff
@@ -40,22 +40,27 @@ func (daemon *Daemon) update(name string, hostConfig *container.HostConfig) erro
 		return err
 	}
 
-	restoreConfig := false
 	backupHostConfig := *ctr.HostConfig
+
+	restoreConfig := false
```

Comment:
Why was this moved?

Reply:
This change can be dropped. It just my style - make variables declaration as near as possible to first variable usage.